src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -27,10 +31,23 @@ private Lift() {
         rightLiftMotor.setIdleMode(IdleMode.kCoast);
         rightLiftMotor.follow(leftLiftMotor);
 
+        leftServo.setBoundsMicroseconds(2000, 1800, 1500, 1200, 1000);
+        rightServo.setBoundsMicroseconds(2000, 1800, 1500, 1200, 1000);
+
         leftLiftMotor.burnFlash();
         rightLiftMotor.burnFlash();
     }
 
+    public void enableRatchets() {
+        leftServo.set(1);
+        rightServo.set(1);
```

Comment:
Set constants for these values instead of explicitly passing them into the "set" method.

Reply:
I agree. What does 1 mean? I know it means full extension but does everyone know that?